comments-bundle/src/Resources/contao/dca/tl_comments.php

```diff
@@ -387,16 +386,18 @@ protected function isAllowedToEditComment($intParent, $strSource)
 			return true;
 		}
 
+		static $cache = array();
```

Comment:
This is considered bad practice (see https://twitter.com/s_bergmann/status/1182282457011363842). If we go this way, it should be a static class property. However, I am not convinced about the change.